packages/utils/cli/src/scan/loadConfig.mjs

```diff
@@ -0,0 +1,32 @@
+import { existsSync } from 'fs'
+import merge from 'lodash.merge'
+
+import * as defaultConfig from './config.mjs'
+
+export async function loadConfig(configFileRoute, { logger }) {
+  try {
+    if (existsSync(configFileRoute)) {
+      logger.info('ℹ️ Loading spark-ui custom configuration file')
+      const { default: customConfig } = await import(configFileRoute)
+
+      const config = {
+        adoption: merge(defaultConfig, {
+          ...customConfig.adoption,
```

Comment:
After testing, this doesn't seem to work, with a custom file using the same structure than `./config.mjs` :confused: Happy to discuss :+1: